src/Find.js

```diff
@@ -1,60 +1,46 @@
 import React, { Component } from 'react'
-import CardFront from './CardFront'
-// import onePlayer from './SinglePlayerData.json'
-// import allPlayers from './WholeTeamData.json'
+import Select from 'react-select'
 import './Find.css'
 
 class Find extends Component {
     constructor() {
         super()
         this.state = {
-            playerData: {}
+            chosenPlayer: {},
+            playerID: null
         }
     }
 
     handleChange = (event) => {
-        this.setState({ playerName: event.target.value })
+        this.props.showFrontOrBack('front')
+        this.setState({ playerID: event.value })
+        const chosenPlayer = this.getSinglePlayerData(event)
+        this.props.setCurrentPlayer(chosenPlayer)
     }
     
-    handleClick = (event) => {
-        event.preventDefault()
-        this.getSinglePlayerData()
-    }
-
-    componentDidMount() {
-
-    }
-    
-    getSinglePlayerData = () => {
-        const lastName = this.state.playerName.split(',')
-        const thisPlayer = this.props.players.find(player => player.LastName === lastName[0])
-        this.setState({ playerData: thisPlayer })
+    getSinglePlayerData(event) {
+        const chosenPlayer = this.props.players.find(player => {
+            return player.PlayerID === Number(event.value)
+        })
+        this.setState({ chosenPlayer: chosenPlayer })
+        return chosenPlayer
```

Comment:
I'm also not sure that I have to put the chosenPlayer into state here, since I'm actually holding it in state in App. Actually maybe this .find should be in App ...

Again, working at the moment; bigger fish to fry.